dask/dataframe/tests/test_dataframe.py

```diff
@@ -6308,6 +6309,20 @@ def test_enforce_runtime_divisions():
             ddf.enforce_runtime_divisions().compute()
 
 
+def test_preserve_ts_unit_in_meta_creation():
+    pdf = pd.DataFrame(
+        {
+            "a": [1],
+            "timestamp": pd.Series(
+                [pd.Timestamp.utcnow()], dtype="datetime64[us, UTC]"
+            ),
+        }
+    )
+    df = dd.from_pandas(pdf, npartitions=1)
+    dd.assert_eq(meta_nonempty(df._meta).dtypes, pdf.dtypes)
+    dd.assert_eq(df, pdf)
```

Comment:
Minor non-blocking nit: `assert_eq` is already imported at the top and used throughout -- probably makes sense do the same here. 

```suggestion
    assert_eq(meta_nonempty(df._meta).dtypes, pdf.dtypes)
    assert_eq(df, pdf)
```